presto-docs/src/main/sphinx/sandbox-readmes/readme-hdp.rst

```diff
@@ -5,34 +5,34 @@ Readme for HDP VM Sandbox
 This is a VirtualBox sandbox image that can be used to experiment with the Presto distribution.
 The credentials for the VM are:
 
-    |  user: **root**
+    |  user: **presto**
     |  password: **presto**
 
 The image is based on Hortonworks HDP 2.2.4 and contains following components:
```

Comment:
I think we're running HDP 2.3, but since we don't mention the Cloudera version in the CDH readme, it might be better to leave out the HDP version.